src/core_modules/capture-core/components/WidgetEventNote/WidgetEventNote.epics.ts

```diff
@@ -77,10 +105,10 @@ export const addNoteForEventEpic = (action$: InputObservable, store: ReduxStore,
             });
         }));
 
-export const removeNoteForEventEpic = (action$: InputObservable) =>
+export const removeNoteForEventEpic = (action$: any) =>
```

Comment:
In `global.ts`, update the definition of `EpicAction` to include `meta`. The code should look like this:
`export type EpicAction<PayloadType, MetaType = Record<string, unknown>> = Observable<Action & { payload: PayloadType, meta: MetaType }>;`

Import and use the global type `EpicAction`. Have a look at how it is used in `OrgUnitFetcher.epics.ts`. The code will look like this: `action$: EpicAction<any, RemoveNoteActionMeta>`